content/docs/deploy/enterprise/changelog.mdx

```diff
@@ -7,6 +7,18 @@ toc_max_heading_level: 5
 
 # Changelog
 
+## 0.23.0
+
+### New
+
+- [Set Request Headers](/docs/reference/routes/headers#set-request-headers) has a new token substitution value that sends the client certificate fingerprint (downstream mTLS must be enabled) to the upstream application or service.
+- [Access Log Fields](/docs/reference/access-log-fields) and [Authorize Log Fields](/docs/reference/authorize-log-fields) settings make it easier for you to review HTTP request logs from the Proxy and Authorize services.
```

Comment:
These settings let you customize the existing logs... maybe we could tweak this description a little to clarify that?

```suggestion
- [Access Log Fields](/docs/reference/access-log-fields) and [Authorize Log Fields](/docs/reference/authorize-log-fields) settings allow you to customize the values that are logged in the access and authorize logs.
```